app-dicts/myspell-it/Manifest

```diff
@@ -1 +1,2 @@
+DIST 874d181c_dict-it.oxt 1373663 BLAKE2B 89748d0dceca6a8f3aba7b76913693ac248f2a5c925e04b76e50f23edebd6b0194da72646837e806882ce6258784d5dfb6c212046f13315479603d953d95b769 SHA512 b48a5ac075b9a20b0370235cb4fc225c91a67e6311eb9e7a96b008ad5e64e56af3daf4b192d38d5892e8d23d5f6e0c962e35f8cfec11a97ed8708e29796e09d3
```

Comment:
Please make the distfile have a version number at least.

Reply:
Done